app/subscriber/src/features/home/utils/makeFilter.ts

```diff
@@ -15,9 +15,11 @@ export const makeFilter = (
 ): ISubscriberContentFilter => {
   const result: ISubscriberContentFilter = {
     actions: applyActions(filter),
+    boldKeywords: filter.boldKeywords ?? undefined,
```

Comment:
This doesn't make sense to me.  Using the nullish operator to force `undefined`?  Please explain reasoning?